apps/studio/src/features/editing-experience/components/MoveResourceModal/MoveResourceModal.tsx

```diff
@@ -71,7 +72,7 @@ const MoveResourceContent = withSuspense(
     const parentDest = resourceStack[resourceStack.length - 2]
     const curResourceId = moveDest?.resourceId
     const { data, fetchNextPage, hasNextPage, isFetchingNextPage } =
-      trpc.resource.getFolderChildrenOf.useInfiniteQuery(
+      trpc.resource.getChildrenOf.useInfiniteQuery(
```

Comment:
issue(perf): I think we should still continue using `getFolderChildrenOf`, otherwise it seems a bit heavy for the client browser to perform the filtering on the entire list of resources, rather than letting the backend perform an initial filtering to folder-like resources.

Reply:
sounds good!

Reply:
went back to using `getFolderChildrenOf`